app/models/host.rb

```diff
@@ -196,19 +196,32 @@ class Host < ApplicationRecord
   supports_not :enable_vmotion
   supports_not :disable_vmotion
   supports_not :vmotion_enabled
+  supports     :ipmi do
+    reason   = _("The Host is not configured for IPMI")   if ipmi_address.blank?
+    reason ||= _("The Host has no IPMI credentials")      if authentication_type(:ipmi).nil?
+    reason ||= _("The Host has invalid IPMI credentials") if authentication_userid(:ipmi).blank? || authentication_password(:ipmi).blank?
+
+    unsupported_reason_add(:ipmi, reason) if reason
+  end
 
   supports :start do
     valid_states = respond_to?(:vim_power_up_from_standby) ? %w[off standby] : "off"
-    msg = validate_ipmi(valid_states)
+
+    msg   = unsupported_reason(:ipmi) unless supports?(:ipmi)
+    msg ||= _("The Host is not in power state #{valid_states.inspect}") unless Array(valid_states).include?(power_state)
+
     unsupported_reason_add(:start, msg) if msg
   end
+
   supports :stop do
-    msg = validate_ipmi("on")
+    msg   = unsupported_reason(:ipmi) unless supports?(:ipmi)
+    msg ||= _("The Host is not in power state #{valid_states.inspect}") unless power_state == "on"
```

Comment:
I am torn between defining `valid_states` and putting `\"on\"` into the error message

Reply:
Ah yeah bad copy&paste from above thanks